assets/src/service/api.js

```diff
@@ -2,6 +2,6 @@ import useFetch from '../hooks/useFetch'
 
 export const useGradeData = (currentCourseId) => useFetch(`http://localhost:5001/api/v1/courses/${currentCourseId}/grade_distribution`)
 export const useAssignmentPlanningData = (currentCourseId, assignmentFilter) => useFetch(`http://localhost:5001/api/v1/courses/${currentCourseId}/assignments?percent=${assignmentFilter}`)
-export const useFilesAccessedData = (currentCourseId, startWeek, endWeek, gradeRange) => useFetch(`http://localhost:5001/api/v1/courses/17700000000${currentCourseId}/file_access_within_week?week_num_start=${startWeek}&week_num_end=${endWeek}&grade=${gradeRange}`)
-export const useInfoData = (currentCourseId) => useFetch(`http://localhost:5001/api/v1/courses/17700000000${currentCourseId}/info`)
```

Comment:
@Adonais0 I fixed the long canvas_Id issue with files. just do a `git pull` or `git fetch` to pull in the changes from the PR

Reply:
perfect, thanks!